x-pack/plugins/ml/server/models/data_recognizer/modules/sample_datasets_flights/ml/mean_delay_by_dest_weather.json

```diff
@@ -0,0 +1,39 @@
+
+{
+  "groups": [],
+  "description": "Kibana Sample Datasets Flights : detect anomalies in mean flight delay",
+  "analysis_config" : {
+    "bucket_span": "1h",
+    "detectors": [
+      {
+        "detector_description": "Mean flight delay time",
+        "function": "mean",
+	"field_name": "FlightDelayMin",
+	"partition_field_name": "DestWeather"
+      }
+    ],
+    "influencers": ["Carrier", "DestWeather"]
+  },
+  "analysis_limits": {
+    "model_memory_limit": "10mb"
+  },
+  "data_description": {
+    "time_field": "timestamp"
+  },
+  "model_plot_config": {
+    "enabled": true
+  },
+  "custom_settings": {
+    "created_by": "ml-module-sample",
+    "custom_urls": [
+      {
+        "url_name": "Raw data",
+        "url_value": "kibana#/discover?_g=(time:(from:\u0027$earliest$\u0027,mode:absolute,to:\u0027$latest$\u0027))&_a=(index:d3d7af60-4c81-11e8-b3d7-01146121b73d)" 
+	},	
+      {
+        "url_name": "Data dashboard",
+        "url_value": "kibana#/dashboard/7adfa750-4c81-11e8-b3d7-01146121b73d?_g=(time:(from:\u0027$earliest$\u0027,mode:absolute,to:\u0027$latest$\u0027))&_a=(filters:!(),query:(language:kuery,query:\u0027\u0027))" 
```

Comment:
As above, IMO it would be useful to pass the `DestWeather` to the target page, in the filters section if possible.